src/ProductBundle/Controller/ProductVariationAdminController.php

```diff
@@ -36,8 +42,15 @@ public function createAction(Request $request = null)
             throw new AccessDeniedException();
         }
 
+        // NEXT_MAJOR: Keep FQCN when bumping Symfony requirement to 2.8+.
+        if (method_exists(\Symfony\Component\Form\AbstractType::class, 'getBlockPrefix')) {
```

Comment:
It is better, but please use a use statement too

Reply:
Done